horace_core/sqw/@Experiment/Experiment.m

```diff
@@ -57,22 +57,33 @@
                 end
             elseif nargin==3
```

Comment:
This may be better and more clear as a `switch-case` rather than a chain of `if`s

Reply:
unclear if you mean the nargin if or the subordinate ifs below the 2 allowed values. For the former, only 2 ifs plus else. For the latter, it's type checking, not sure if switch supports. Nice thought but let's leave it for now.

Reply:
Referring to `nargin` where you have 0, 1, 3, other, but it's true it's non-essential.